gen/operations_portal_egi

```diff
@@ -6,11 +6,11 @@ use perunServicesUtils;
 use File::Basename;
 
 our $SERVICE_NAME = basename($0);
-our $PROTOCOL_VERSION = "3.0.0";
+our $PROTOCOL_VERSION = "3.0.1";
```

Comment:
Careful, this is `$PROTOCOL_VERSION` and not `$SCRIPT_VERSION`. You haven't changed the protocol as script provides same data structure. Script seems to be missing `$SCRIPT_VERSION` so you can add it and set it to `3.0.0`.

Reply:
I see, thanks. Fixed.